app/src/main/java/ca/pkay/rcloneexplorer/Services/SyncService.java

```diff
@@ -164,6 +180,24 @@ protected void onHandleIntent(@Nullable Intent intent) {
         stopForeground(true);
     }
 
+    public static String humanReadableBytes(long bytes) {
```

Comment:
Use the builtin https://developer.android.com/reference/android/text/format/Formatter#formatFileSize(android.content.Context,%20long)

We could also factor [this](https://github.com/x0b/rcx/blob/fbb5ad39f241b198f078da36366b101e8baac05d/app/src/main/java/ca/pkay/rcloneexplorer/Items/FileItem.java#L123-L129) into a Formatter class. [That may not be the best idea though 😂](https://programming.guide/worlds-most-copied-so-snippet.html), better use the Android Formatter.

Reply:
I have searched for a class that would do the conversion for me, but i couldn't find it! Thanks for pointing this out!